Ladybird/Settings.cpp

```diff
@@ -8,9 +8,9 @@
 
 namespace Browser {
 
-Settings::Settings(QObject* parent)
+Settings::Settings()
 {
-    m_qsettings = new QSettings("Serenity", "Ladybird", parent);
+    m_qsettings = new QSettings("Serenity", "Ladybird");
```

Comment:
ah, one final thing. Now that `Browser::Settings` is not owned by a parent QObject, if we don't attach a 'parent' to this QSettings object, it will be leaked when `Browser::Settings` is destroyed. The simplest fix is to pass `this` in place of `parent`.

Reply:
Done but I had to keep the `Settings` a `QObject` :)